src/frequenz/dispatch/_actor_dispatcher.py

```diff
@@ -225,18 +173,18 @@ def __init__(  # pylint: disable=too-many-arguments, too-many-positional-argumen
             running_status_receiver: The receiver for dispatch running status changes.
             dispatch_identity: A function to identify to which actor a dispatch refers.
                 By default, it uses the dispatch ID.
-            retry_interval: The interval between retries.
+            retry_interval: How long to wait until trying to start failed actors again.
         """
         super().__init__()
         self._dispatch_identity: Callable[[Dispatch], int] = (
             dispatch_identity if dispatch_identity else lambda d: d.id
         )
 
         self._dispatch_rx = running_status_receiver
+        self._retry_timer_rx = Timer(retry_interval, SkipMissedAndDrift())
         self._actor_factory = actor_factory
         self._actors: dict[int, ActorDispatcher.ActorAndChannel] = {}
-
-        self._retrier = ActorDispatcher.FailedDispatchesRetrier(retry_interval)
+        self._pending_dispatches: dict[int, Dispatch] = {}
```

Comment:
Oww I god it!
Than Maybe rename it to _failed_dispatches? `pending` feels like they just started and didn't finished.